data/sql/updates/pending_db_world/rev_1674777294431128400.sql

```diff
@@ -0,0 +1,16 @@
+--
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (-58200, -58201, -58202, -58203) AND (`source_type` = 0) AND (`id` IN (0));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-58200, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36'),
+(-58201, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36'),
+(-58202, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36'),
+(-58203, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36');
```

Comment:
Shattered Hand Captains are assigned a SmartAI. Wouldn't this override it and prevent them from casting spells?

Reply:
There could be a workaround for that by using waypoint_scripts. Set a stationary waypoint with a delay, make them emote every time this waypoint is "reached"
That's the best workaround I found so far :p

Reply:
> Shattered Hand Captains are assigned a SmartAI. Wouldn't this override it and prevent them from casting spells?

Guids sai will take priority over entey sai